### ISSUE TITLE
Add shields to README for code coverage

### ISSUE THREAD

**Issue by bbriggs** [voxpupuli/plumbing#84]

**Comment by dhoppe** [comment]
This requires modulesync_config version 0.14.1, because previous versions does not have support for Coveralls.

- [ ] puppet-alternatives
- [ ] puppet-amanda
- [ ] puppet-archive
- [ ] puppet-collectd
- [ ] puppet-community_kickstarts
- [ ] puppet-confluence
- [ ] puppet-corosync
- [ ] puppet-dhcp
- [ ] puppet-dotnet
- [ ] puppet-download_file
- [ ] puppet-drbd
- [ ] puppet-etherpad
- [ ] puppet-extlib
- [ ] puppet-fetchcrl
- [ ] puppet-filemapper
- [ ] puppet-gerrit
- [ ] puppet-ghost
- [ ] puppet-gluster
- [ ] puppet-googleauthenticator
- [ ] puppet-graphite_powershell
- [ ] puppet-healthcheck
- [ ] puppet-hiera
- [ ] puppet-iis
- [ ] puppet-jenkins_job_builder
- [ ] puppet-jira
- [ ] puppet-kafka
- [ ] puppet-mcollective
- [ ] puppet-minecraft
- [ ] puppet-mrepo
- [ ] puppet-msoffice
- [ ] puppet-mumble
- [ ] puppet-mysql_java_connector
- [ ] puppet-network
- [ ] puppet-nginx
- [ ] puppet-nodejs
- [ ] puppet-nsclient
- [ ] puppet-puppetboard
- [ ] puppet-rhsm
- [ ] puppet-rundeck
- [ ] puppet-selinux
- [ ] puppet-sftp_jail
- [ ] puppet-splunk
- [ ] puppet-squid
- [ ] puppet-sslcertificate
- [ ] puppet-staging
- [ ] puppet-stash
- [ ] puppet-unattended_upgrades
- [ ] puppet-visualstudio
- [ ] puppet-windows_autoupdate
- [ ] puppet-windows_eventlog
- [ ] puppet-windows_firewall
- [ ] puppet-windows_power
- [ ] puppet-windowsfeature
- [ ] puppet-yum
- [ ] puppet-zabbix

**Comment by bbriggs** [comment]
Have we been updating the list of modules that need badges when we import new things? I know we've picked up a handful recently

**Comment by dhoppe** [comment]
@bbriggs No, we did not? But this should only be a diff against the list from  modulesync_config.

**Comment by bbriggs** [comment]
I'll do that real quick.

**Comment by bastelfreak** [comment]
If you write a script that updates a README.md, we can throw that into modulesync_config. As pointed out, modulesync doesn't manage the README.mdd (although we discussed that in the past). But modulesync_config has the list of all maintained modules and it has secrets that allow write access to all tose modules to raise pull requests. So we could run such an update script in a github  workflow.

---

If you want to make this a bit more challenging: there are a few more labels which we usually add. My goto example is always https://github.com/voxpupuli/puppet-openssl?tab=readme-ov-file#openssl-puppet-module .

**Comment by TheMeier** [comment]
well here is the full script:

```
sed -E 's#\[!\[Build Status\]\(https://travis-ci\.org/voxpupuli/([^.]+)\.svg\?branch=master\)\]\([^)]+\)#![Build Status](https://github.com/voxpupuli/\1/actions/workflows/ci.yml/badge.svg?branch=master)#g' README.md

```
Im not sure how runing that in CI makes much sense, since it is a one time operation...
